Specs/DateTime/English/MergedParser.json

```diff
@@ -1018,5 +995,164 @@
       }
     }
   ]
+},
+{
+  "Input": "I'll change the ten am appointment to twenty!",

```

Comment:
What if there is no punctuation char at the end?

Reply:
the same. We have similar test cases for that.